sites/authentication/authentication-fr.html

```diff
@@ -19,7 +19,7 @@
 
 <h2>Bouton contextuel « Se connecter »</h2>
 
-<p>En définissant les paramètres appropriés, le bouton de connexion contextuel apparaîtra au chargement de la page et sera situé dans le coin supérieur droit de la page sous la ligne horizontale tel qu'indiqué dans les <a href="https://conception.canada.ca/configurations-conception-communes/connexion-contextuel.html">lignes directrices du système de conception de Canada.ca</a>.</p>
+<p>En définissant les paramètres appropriés, le bouton de connexion contextuel apparaîtra au chargement de la page et sera situé dans le coin supérieur droit de la page sous la ligne horizontale tel qu'indiqué dans les <a href="https://conception.canada.ca/configurations-conception-communes/connexion-contextuel.html">conseils sur le bouton contextual «&nbsp;Se connecter&nbsp;»</a>.</p>
```

Comment:
@Ricokola Petit typo :
```suggestion
<p>En définissant les paramètres appropriés, le bouton de connexion contextuel apparaîtra au chargement de la page et sera situé dans le coin supérieur droit de la page sous la ligne horizontale tel qu'indiqué dans les <a href="https://conception.canada.ca/configurations-conception-communes/connexion-contextuel.html">conseils sur le bouton contextuel «&nbsp;Se connecter&nbsp;»</a>.</p>
```
(J'ai corrigé le billet aussi.)